charts/core/traefik/templates/middlewares/stripPrefixRegex.yaml

```
@@ -0,0 +1,11 @@
+{{ range $index, $middlewareData := .Values.middlewares.stripPrefixRegex }}
+---
+apiVersion: traefik.containo.us/v1alpha1
+kind: Middleware
+metadata:
+  name: {{ $middlewareData.name }}
+  namespace: {{ $namespace }}
+spec:
+  stripPrefixRegex:
+    regex: {{ $middlewareData.regex | quote }}
+{{ end }}
```

Why did you not-copy the whole important section above this?

```suggestion
{{- $values := .Values }}
{{- $namespace := ( printf "ix-%s" .Release.Name ) }}
{{- if or ( not .Values.ingressClass.enabled ) ( and ( .Values.ingressClass.enabled ) ( .Values.ingressClass.isDefaultClass ) ) }}
{{- $namespace = "default" }}
{{- end }}

{{ range $index, $middlewareData := .Values.middlewares.stripPrefixRegex }}
---
apiVersion: traefik.containo.us/v1alpha1
kind: Middleware
metadata:
  name: {{ $middlewareData.name }}
  namespace: {{ $namespace }}
spec:
  stripPrefixRegex:
    regex: {{ $middlewareData.regex | quote }}
{{ end }}
```

---

Because I read the first commit that added the `charts/core/traefik/templates/middlewares/forwardauth.yaml` file 😅